### PR TITLE
Rule updates for January 2023

### PR DESCRIPTION
**Pull request by p4p3r**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/release-notes/rule-updates.mdx [hunk 1]
```diff
@@ -12,6 +12,73 @@ import MoreHelp from "/src/components/MoreHelp"
...
+## January 2023
+
+### Community rules
+
+#### New rules from Semgrep community and r2c
+
+- Thanks [johnssimon007](https://github.com/johnssimon007)! New rule for empty encryption key in Python: <LinkToRegistryRule ruleId="python.cryptography.security.empty-aes-key.empty-aes-key" />
+- Thanks [johnssimon007](https://github.com/johnssimon007)! New rule for header injection in Python/Flask: <LinkToRegistryRule ruleId="python.flask.security.audit.host-header-injection-python.host-header-injection-python
+" />
+- Additional rule for deserialization vulnerabilities in Ruby: <LinkToRegistryRule ruleId="ruby.lang.security.bad-deserialization-yaml-env.bad-deserialization-yaml-env" />
```
Comment (adamkvitek) [inline docs/release-notes/rule-updates.mdx:24]: @p4p3r not working link

File: docs/release-notes/rule-updates.mdx [hunk 2]
```diff
@@ -12,6 +12,73 @@ import MoreHelp from "/src/components/MoreHelp"
...
 Welcome to monthly rule updates. This document includes selected new rules, removed false positives (FP), and other rule updates. These new rules and their updates are made by the Semgrep community and r2c.
 
+## January 2023
+
+### Community rules
+
+#### New rules from Semgrep community and r2c
+
+- Thanks [johnssimon007](https://github.com/johnssimon007)! New rule for empty encryption key in Python: <LinkToRegistryRule ruleId="python.cryptography.security.empty-aes-key.empty-aes-key" />
+- Thanks [johnssimon007](https://github.com/johnssimon007)! New rule for header injection in Python/Flask: <LinkToRegistryRule ruleId="python.flask.security.audit.host-header-injection-python.host-header-injection-python
```
Comment (adamkvitek) [inline docs/release-notes/rule-updates.mdx:22]: The link displays no results.

Comment (adamkvitek) [reply docs/release-notes/rule-updates.mdx]: Fixed it I think. Resolving.

File: docs/release-notes/rule-updates.mdx (hunk 1 above)
Comment (p4p3r) [reply docs/release-notes/rule-updates.mdx]: Fixed in c5a3bb12c8d183b0b3a743d73ddc9a95d66fd377

Comment (adamkvitek) [reply docs/release-notes/rule-updates.mdx]: Fixed in https://github.com/returntocorp/semgrep-docs/pull/893/commits/c5a3bb12c8d183b0b3a743d73ddc9a95d66fd377

**Review by adamkvitek — APPROVED** [review]

**Pull request merged**